### ISSUE TITLE
Notifications not being received.

### ISSUE THREAD

**Issue by keco1249** [elvismercado/meteor-raix-push-demo#1]
Cloned project, updated config.push.json with android key & project number and removed ios credentials. I then run the project and log in and get no errors but the Push message listener never receives the messages that get sent via Push.send. The database updates and ui shows the messages that get sent but you can see the received column never gets updated nor does the console print out the message string. Am I missing a step or something?

**Comment by malcolmevans** [comment]
Hi @keco1249 - did you ever find a solution for this? Running into the same issue on my end.

**Comment by keco1249** [comment]
Yeah it was just an issue on my dev machine's firewall with incoming/outgoing connections.

**Comment by ajschmaltz** [comment]
I'm having a problem too.  No console message and no received update.  Any tips?
